ui/static_table/viewmodels/OBATableRow.m

```diff
@@ -23,7 +23,7 @@ + (void)load {
 }
 
 + (instancetype)tableRowWithTitle:(NSString*)title action:(void (^)())action {
-    return [[self alloc] initWithTitle:title action:action];
+    return [[OBATableRow alloc] initWithTitle:title action:action];
```

Comment:
I'd be totally ok with this class method going away altogether if you wanted to take care of that :)